src/pybind/mgr/nfs/export.py

```diff
@@ -425,20 +426,21 @@ def list_all_exports(self) -> List[Dict[str, Any]]:
     @export_cluster_checker
     def list_exports(self,
                      cluster_id: str,
-                     detailed: bool = False) -> Tuple[int, str, str]:
+                     detailed: bool = False) -> List[Any]:
```

Comment:
When you say
```
Note that without additional code change both these apis can
reply in both JSON and YAML formats.
```
do you mean we could provide `--format json` on the cli for this command and it would work without us needing to add format as a param for the function?

Reply:
Yes, that is the intent (yaml too).